source/TR-181/middle_layer_src/wanmgr_dml_iface_apis.c

```diff
@@ -1046,8 +1050,16 @@ BOOL WanIfCfg_SetParamBoolValue(ANSC_HANDLE hInsContext, char* ParamName, BOOL b
             }
             if (strcmp(ParamName, "EnableIPoEHealthCheck") == 0)
             {
-                pWanDmlIface->VirtIfList->EnableIPoE = bValue;
-                ret = TRUE;
+                CONNECTIVITY_CHECK_TYPE type = bValue? WAN_CONNECTIVITY_TYPE_IHC:WAN_CONNECTIVITY_TYPE_NO_CHECK;
+                if((type != pWanDmlIface->VirtIfList->IP.ConnectivityCheckType) &&
```

Comment:
Please add a comment for the if conditions

Reply:
okay